ztp/siteconfig-generator/siteConfig/siteConfig.go

```diff
@@ -236,6 +247,7 @@ type Nodes struct {
 	IgnitionConfigOverride string                 `yaml:"ignitionConfigOverride"`
 	Role                   string                 `yaml:"role"`
 	CrTemplates            map[string]string      `yaml:"crTemplates"`
+	DiskPartition		   []DiskPartition        `yaml:"diskPartition"`
```

Comment:
It seems like this would be forcing users to define disk partitions uniquely on every node; is that right?  For 3-node and standard clusters, it seems like they'd probably want to define a disk partition and have it appear on every worker...

Reply:
One thing though...I'm assuming this is valid only for single node. But regardless of however many nodes...to create the partition on a node, the user must know its underlying hardware metadata including device path and available space which maybe different for every node. 

Also, in this case, once a partition is made only a single PV and PVC (can't use local storage operator with partition) is made which is the link between the image registry and disk..i.e even if you have multiple partitions user can't really leverage the other partitions on other nodes. 

I guess the bottom line is...users can make as many partitions as they like on however many nodes...only one is ever being is used for image registry. So the question is do we limit the ability to create partitions?

Reply:
As discussed, this feature is only for SNO and ok for it stay under `nodes` list.